content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+
+Lihat juga [dokumentasi pembaruan](/id/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15).
+
+{{< caution >}}
+Laman ini tidak menunjukkan cara untuk menjalankan klastermu pada penyedia layanan _cloud_. Pada _environment cloud_, kedua pendekatan yang didokumentasikan di sini tidak akan bekerja untuk objek Service dengan tipe LoadBalancer maupun PersistentVolume dinamis.
+{{< /caution >}}
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+Untuk kedua metode kamu membutuhkan infrastruktur seperti berikut:
+
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk
+  Node _master_
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk Node _worker_
+- Konektivitas internet pada seluruh mesin di dalam klaster (baik jaringan publik maupun jaringan pribadi)
+- Hak akses sudo pada seluruh mesin
+- Akses SSH dari satu perangkat ke seluruh Node pada sistem
+- Perkakas `kubeadm` dan `kubelet` diinstal pada seluruh mesin. Perkakas `kubectl` bersifat opsional.
+
+Untuk klaster etcd eksternal saja, kamu juga membutuhkan:
+
+- Tiga mesin tambahan untuk anggota-anggota etcd
+
+
+
+<!-- steps -->
+
+## Langkah pertama untuk kedua metode
+
+### Membuat _load balancer_ untuk kube-apiserver
+
+{{< note >}}
+Akan ada banyak konfigurasi untuk _load balancer_. Contoh berikut ini hanyalah salah satu
+opsi. Kebutuhan klastermu mungkin membutuhkan konfigurasi berbeda.
+{{< /note >}}
+
+1.  Buat sebuah _load balancer_ kube-apiserver dengan sebuah nama yang yang akan mengubah ke dalam bentuk DNS.
+
+    - Pada _environment cloud_ kamu harus meletakkan Node _control plane_ di belakang _load balancer_ yang meneruskan TCP. _Load balancer_ ini mendistribusikan trafik ke seluruh Node _control plane_ pada daftar tujuan. _Health check_ untuk
+      apiserver adalah pengujian TCP pada porta yang didengarkan oleh kube-apiserver
+      (nilai semula `:6443`).
+
+    - Tidak direkomendasikan untuk menggunakan alamat IP secara langsung pada _environment cloud_.
+
+    - _Load balancer_ harus dapat berkomunikasi dengan seluruh Node _control plane_ 
+      pada porta yang digunakan apiserver. _Load balancer_ tersebut juga harus mengizinkan trafik masuk pada porta yang didengarkannya.
+
+    - Pastikan alamat _load balancer_ sesuai
+      dengan alamat `ControlPlaneEndpoint` pada kubeadm.
+
+    - Baca panduan [Opsi untuk _Software Load Balancing_](https://github.com/kubernetes/kubeadm/blob/master/id/docs/ha-considerations.md#options-for-software-load-balancing)
+      untuk detail lebih lanjut.
+
+2.  Tambahkan Node _control plane_ pertama pada _load balancer_ dan lakukan pengujian koneksi:
+
+    ```sh
+    nc -v LOAD_BALANCER_IP PORT
+    ```
+
+    - Kegalatan koneksi yang ditolak memang diantisipasi karena apiserver belum
```

Comment:
oh nice one https://kbbi.kemdikbud.go.id/entri/galat

Reply:
Yep! It's usually used to translate "error".